File: Data Structures & Algorithms/two-integer-sum/submission-1.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        freq={}
        res=[]
        sorted(nums)
        for i in range(len(nums)):
            if target - nums[i] in freq:
                res.append(freq[target-nums[i]])
                res.append(i)
            else:
                freq[nums[i]]=i
        return res

File: Data Structures & Algorithms/majority-element/submission-1.java
class Solution {
    public int majorityElement(int[] nums) {
        int candidate = nums[0], ctr=1;
        for(int ii : nums) {
            if(ctr == 0) {
                candidate = ii;
                ctr=1;
            } else if(ii == candidate) {
                ctr++;
            } else {
                ctr-=1;
            }
        }
        return candidate;

    }
}